Src/Eiffel/eiffel/interface/feature_i.e

```diff
@@ -2313,6 +2313,16 @@ feature -- Signature checking
 				then
 					error_handler.insert_error (create {VRVA}.make_invalid_type (Current, a_context_class, written_class, l_type))
 				end
+					-- A generic class can't have a once creation procedure
+				if is_once and then a_context_class.is_generic then
+					error_handler.insert_error (create {VGCC10}.make_invalid_context (Current, a_context_class, written_class))
```

Comment:
How do we know that this feature is a creation procedure?

Reply:
Good point I need to check it's is a once creation procedure.

Reply:
I'd suggest to add eweasel tests for all validity rules.